### ISSUE TITLE
yaml warning from feediverse

### ISSUE THREAD

**Issue by steinarb** [edsu/feediverse#15]
When being run feediverse outputs the following warning
```
/usr/local/lib/python3.7/dist-packages/feediverse.py:48: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  config = yaml.load(fh)
```

**Comment by htgoebel** [comment]
This is already fixed in https://github.com/edsu/feediverse/pull/8

**Comment by edsu** [comment]
This should be fixed since #8 has been applied. Thanks!
